make/autoconf/flags-cflags.m4

```diff
@@ -574,7 +574,7 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
 
   # CXXFLAGS C++ language level for all of JDK, including Hotspot.
   if test "x$TOOLCHAIN_TYPE" = xgcc || test "x$TOOLCHAIN_TYPE" = xclang || test "x$TOOLCHAIN_TYPE" = xxlc; then
-    LANGSTD_CXXFLAGS="-std=c++14"
+    LANGSTD_CXXFLAGS="-std=gnu++14"
```

Comment:
Uhhh, is this really okay? I thought the idea was that we should standardize on official C++, not use vendor-specific extensions. 

@kimbarrett Opinions on this?

Reply:
I pulled it out. The problem is that I have to do this:

```
    jdouble_cast(0x0030000000000000); // 0x1.0p-1020;
```

which is yucky, but not the End Of The World.